crates/tauri/src/manager/mod.rs

```diff
@@ -127,7 +128,7 @@ fn replace_csp_nonce(
   directive: &str,
   hashes: Vec<String>,
 ) {
-  let mut nonces = Vec::new();
+  let mut nonces = Vec::with_capacity(asset.matches(token).count());
   *asset = replace_with_callback(asset, token, || {
     #[cfg(target_pointer_width = "64")]
     let mut raw = [0u8; 8];
```

Comment:
@lucasfernog do you remember why we used `usize` here at the first place (tracing back to https://github.com/tauri-apps/tauri/commit/cf54dcf9c81730e42c9171daa9c8aa474c95b522)? It doesn't quite make sense to me

Reply:
I meant it doesn't make sense to make nonce length depends on target pointer size